templates/vagrantfiles/Vagrantfile.ad_master_2client

```diff
@@ -5,7 +5,6 @@
 Vagrant.configure(2) do |config|
```

Comment:
Could you add a comment with recommended number of CPUs and amount of memory required for this topology?

Reply:
I've included in the top of the mentioned topologies the following:
```ruby
# Add the following to the topologies section of your test definitions:
#
# ad_master_2client: &ad_master_2client
#   name: ad_master_2client
#   cpu: 4
#   memory: 12000
```